functions/Invoke-DbaAdvancedRestore.ps1

```diff
@@ -82,6 +82,9 @@ function Invoke-DbaAdvancedRestore {
     .PARAMETER Confirm
         Prompts you for confirmation before running the cmdlet.
 
+    .PARAMETER RestoreAsSA
```

Comment:
Since it is part of the restore commands maybe we call this just `AsSA` or maybe more appropriate to the solution `ExecuteAsSa`?

Reply:
ExecuteAsSa sounds better to me.

Reply:
changed and pushed back up